cassandra-migration/src/main/resources/db/migration/cassandra/V1.20_cassandra.cql

```diff
@@ -7,3 +7,6 @@ INSERT INTO sunbird.email_template(name, template) VALUES ('welcome', '<html> <h
 
+
+CREATE KEYSPACE IF NOT EXISTS qmzbm_form_service WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 }
```

Comment:
Can you add these changes in last CQL file of release 1.11 (i.e. V1.26_cassandra.cql)?